src/data/builds/barbarian/2001_barb_min_100.ts

```diff
@@ -4,8 +4,8 @@ import { Build } from "~/composables/Builds";
 
 export const build: Build = {
   title: "Mining Active / AFK (Barb) 100+",
-  notes: "",
-  version: "1.23",
+  notes: "Use Icing Ironbites, Golden Peanuts and Life Potions. Mining Event food should generally be preserved for sampling. Do not level your mining skill higher than the highest tool.",
```

Comment:
It's not "obvious" at a glance why someone shouldn't level their mining (and technically I don't think it's fair advice, as at higher efficiency levels provide a good boost).

What do you think about potentially just recommended to keep it lower than mman for the efficiency boost? 

Unless I misunderstood the comment :)